boot-qemu.py

```diff
@@ -187,6 +199,9 @@ def run(self):
                 utils.red("ERROR: QEMU did not exit cleanly!")
             sys.exit(err.returncode)
 
+    def supports_efi(self):
+        return False
```

Comment:
Does this really need to be a method? Could it be a member of the base class that the derived classes modify in their constructors?

Reply:
No, definitely not. Done:

b845151d1e2e9c1296e0aaa83a4cc4dae861c5e5